sql/item_weapon.sql

```diff
@@ -3975,12 +3975,54 @@ INSERT INTO `item_weapon` VALUES (21056,'tomonari',10,0,0,0,0,2,1,420,118,0);
 INSERT INTO `item_weapon` VALUES (21057,'tomonari_+1',10,0,0,0,0,2,1,407,119,0);
 INSERT INTO `item_weapon` VALUES (21058,'shichishito',10,0,54,54,42,2,1,450,126,0);
 INSERT INTO `item_weapon` VALUES (21059,'ichijinto',10,0,0,0,0,2,1,420,121,0);
+INSERT INTO `item_weapon` VALUES (21060,'mjollnir',11,0,242,242,228,3,1,308,160,0);
 INSERT INTO `item_weapon` VALUES (21061,'mjollnir',11,0,242,242,228,3,1,308,160,0);
-INSERT INTO `item_weapon` VALUES (21062,'yagrush',11,0,242,242,228,3,2,267,133,0);
-INSERT INTO `item_weapon` VALUES (21063,'yagrush',11,0,242,242,228,3,2,267,133,0);
+INSERT INTO `item_weapon` VALUES (21062,'yagrush',11,0,242,242,228,3,3,267,133,0);
```

Comment:
p.s. 
You seem to be merging dsp into your fork to update. Set DSP as a remote and just pull from DSP master instead, and then you won't get all these merges in your history showing on your pull requests. 

It also pays to use branches other than master to make pull requests with, and leave master 100% unchanged from stock DSP, since by default new branches are 1:1 copies of your master branch. This is how I so often have 3-4 pull requests open at once with different things in them.